### PR TITLE
Upgrade package with CG vulnerability

### PR DESCRIPTION
**Pull request by jekloudaMSFT**
For more information about how to contribute to this repo, visit [this page](https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md).

## Description

This package was flagged on the Android pipeline. Upgrading to fix the CG alert

### CODE REVIEW

**Review by jadahiya-MSFT — APPROVED** [review]

File: apps/teamsjs-diagnostic-app/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "@microsoft/teamsfx": "^2.3.2",
     "@microsoft/teamsfx-react": "^3.1.2",
     "@testing-library/dom": "^10.4.0",
-    "axios": "^1.7.3",
+    "axios": "^1.7.4",
```
Comment (AE-MS) [inline apps/teamsjs-diagnostic-app/package.json:14]: Should an update to `pnpm-lock.yaml` come in this same change?

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

File: apps/teamsjs-diagnostic-app/package.json (hunk 1 above)
Comment (jekloudaMSFT) [reply apps/teamsjs-diagnostic-app/package.json:14]: Yep, just pushed

**Review by jadahiya-MSFT — APPROVED** [review]

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -3632,8 +3632,8 @@ packages:
   axios@0.28.1:
```
Comment (AE-MS) [inline pnpm-lock.yaml:3632]: >axios@0.28.1 [](http://example.com/codeflow?start=2&length=12)

For others who might look at this later, it is okay that we still depend on axios 0.28.1 as the vulnerable versions only start with 1.x. So 0.28.1 is okay (at least as far as this particular vulnerability is concerned.)

**Review by AE-MS — APPROVED** [review]
:shipit:

**Pull request merged**